src/main/sensors/battery.h

```diff
@@ -76,3 +84,5 @@ int32_t getAmperageLatest(void);
 int32_t getMAhDrawn(void);
 
 void batteryUpdateCurrentMeter(timeUs_t currentTimeUs);
+
+lowVoltageCutoff_t getLowVoltageCutoff(void);
```

Comment:
Returning struct is not that common in C and in CF ... 

Also note the missing newline